Eugene-CG/popup/index.html

```diff
@@ -0,0 +1,52 @@
+    <title>Popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <div class="log">
+                Logo
+            </div>
+            <nav class="menu">
+                <ul class="list">
+                    <li class="list__item"><a href="#">Mail</a></li>
+                    <li class="list__item"><a href="#">Images</a></li>
+                    <li class="list__item popup">
+                        <label for="popup-btn__label">
+                            <img src="./img/popup-button.png" alt="">
+                        </label>
+                        <input type="checkbox" class="popup-btn" id="popup-btn">
+                        <ul class="popup-list">
+                            <li class="popup-list__item"><a href="#" class="popup-list__link"><img src="./img/blogs.png" alt="blogs"></a></li>
+                            <li class="popup-list__item"><a href="#" class="popup-list__link"><img src="./img/calendar.png" alt="calendar"></a></li>
+                            <li class="popup-list__item"><a href="#" class="popup-list__link"><img src="./img/chat.png" alt="chat"></a></li>
+                            <li class="popup-list__item"><a href="#" class="popup-list__link"><img src="./img/classroom.png" alt="classroom"></a></li>
+                            <li class="popup-list__item"><a href="#" class="popup-list__link"><img src="./img/collection.png" alt="collection"></a></li>
+                            <li class="popup-list__item popup-more">
+                                <label for="more-btn__label">
+                                    <span class="more-btn__span">More</span>
+                                </label>
+                                <input type="checkbox" class="more-btn" id="more-btn">
+                                <ul class="more-list">
+                                    <li class="more-list__item"><a href="#" class="more-list__link"><img src="./img/contacts.png" alt="contacts"></a></li>
+                                    <li class="more-list__item"><a href="#" class="more-list__link"><img src="./img/documents.png" alt="documents"></a></li>
+                                    <li class="more-list__item"><a href="#" class="more-list__link"><img src="./img/favicon.png" alt="favicon"></a></li>
+                                    <li class="more-list__item"><a href="#" class="more-list__link"><img src="./img/files.png" alt="files"></a></li>
+                                </ul>
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="menu__list-item"><a href="#"><img src="./img/bell.png" width="50px" height="45px" alt="Bell"></a></li>
+                    <li class="menu__list-item cheshire"><a href="#"><img src="./img/cheshire.jpg" width="50px" height="45px" alt="Avatar Icon"></a></li>
```

Comment:
It is really great that you have fount that attributes width and height could be passed to img tag. But they are using, when css styles are don't. So if you use css styles this attributes using is incorrect for html and css separation logic.
Please, move this styles into css.
![image](https://user-images.githubusercontent.com/42061361/181093025-d2c7b346-1e52-4026-88af-7f6dcf4e5d00.png)

Reply:
Sorry, bro. Close this issue. Have serched about this  and this is my bad. Please revert  all  changes related to this comment.
https://jakearchibald.com/2022/img-aspect-ratio/

Reply:
All img attributes removed from html to css.

Reply:
You should add alt attributes for img tag, but it should be empty without any white spaces or symbols.  Example `<img src="./some_path" alt="" >`